frontend/containers/project-page/impact/component.tsx

```diff
@@ -122,6 +125,32 @@ export const Impact: React.FC<ImpactProps> = ({ project, enums }: ImpactProps) =
                 >
                   <FormattedMessage defaultMessage="How is the impact calculated?" id="9cE0nR" />
                 </button>
+                <div
+                  className={cx('transition-all ease-in-out duration-300', {
+                    'h-0 opacity-0': impactLocation !== ImpactAreas.PriorityLandscape,
+                    'h-auto opacity-100': impactLocation === ImpactAreas.PriorityLandscape,
+                  })}
+                >
```

Comment:
You could probably add this:
```suggestion
                <div
                  className={cx('transition-all ease-in-out duration-300', {
                    'h-0 opacity-0': impactLocation !== ImpactAreas.PriorityLandscape,
                    'h-auto opacity-100': impactLocation === ImpactAreas.PriorityLandscape,
                  })}
                  aria-hidden={impactLocation !== ImpactAreas.PriorityLandscape}
                >
```